src/pages/Search/EmptySearchView.tsx

```diff
@@ -46,6 +47,15 @@ function EmptySearchView({type}: EmptySearchViewProps) {
     const StyleUtils = useStyleUtils();
     const {translate} = useLocalize();
     const styles = useThemeStyles();
+    const [modalVisible, setModalVisible] = useState(false);
+    const [allPolicies] = useOnyx(ONYXKEYS.COLLECTION.POLICY);
+    const shouldRedirectToExpensifyClassic = useMemo(() => {
```

Comment:
It looks like this is the same logic that exists [here](https://github.com/Expensify/App/blob/bd6c90d8493baf70d61937188c94815650f18988/src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx#L193-L199). Should we move this to a util function instead?